package.json

```diff
@@ -1,38 +1,40 @@
 {
   "name": "komada",
   "version": "0.20.1",
-  "author": "Evelyne Lachance",
   "description": "Komada: Croatian for 'pieces', is a modular bot system including reloading modules and easy to use custom commands.",
-  "main": "index.js",
-  "scripts": {
-    "test": "eslint classes commands finalizers functions inhibitors events client.js",
-    "lint": "npm run test -- --fix"
+  "homepage": "https://github.com/dirigeants/komada#readme",
+  "bugs": {
+    "url": "https://github.com/dirigeants/komada/issues"
   },
+  "license": "ISC",
+  "author": "Evelyne Lachance",
+  "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/dirigeants/komada.git"
   },
-  "bugs": {
-    "url": "https://github.com/dirigeants/komada/issues"
+  "scripts": {
+    "lint": "npm run test -- --fix",
+    "test": "eslint classes commands finalizers functions inhibitors events client.js"
   },
-  "license": "ISC",
   "dependencies": {
     "chalk": "^1.1.3",
     "discord.js": "hydrabolt/discord.js",
     "fs-nextra": "0.0.3",
+    "lpad": "^2.0.1",
```

Comment:
.padEnd() .padStart() are available native in node V8.0.0.

You do not need a package for this functionality...